MoMMI/Modules/chance.py

```diff
@@ -95,3 +95,27 @@
     register_help(__name__, "pick", """Man can you believe this? People actually want to do *fair* 50/50 picks between things? Kids these days.
 
 Fine, just run @MoMMI pick(a,b,c) with as many comma separated values as you need. Normies.""")
+
+# todo
+# support short label codes like qol, bugfix etc, so not search for labels literally
+# filter for emojicracy, just use emoji-modifiers to calc total value of issue, then choose between top ~10ish
+@command("giveissue", r"(?:giveissue)(?:\s+([\w\,]*))?(?:\s+(\w\/))?")
+async def giveissue_command(channel: MChannel, match: Match, message: Message) -> None:
+    channel.send(":hourglass_flowing_sand: Fetching random issue")
+
+    #getting labels
+    t_labels = [x.strip() for x in match.group(1).split(",")] # strip whitespaces
+    params = {"labels" : t_labels.join(",")}
+
+    #getting repo
+    repo = "vgstation-coders/vgstation13"
+    if(match.group(2))
```

Comment:
You didn't put a quantifier on that second capturing group OR use a [] so you can't actually enter anything useful here.

Also this should use repo prefixes.

Reply:
what are repo prefixes if i might ask?

Reply:
They're how you specify which repo to use for issue lookup. Now obviously we only have one repo on /vg/ but for SS14 we have multiple so [c#123] gets a thing from the content repo whereas just [123] will list the issue in both repos.

Reply:
so basically i'd need to add a repo prefix to my issue_command call.

also what prefix does vg use/ how would i generate that prefix from the repo string

Reply:
The prefixes are defined in my config files. Check how the issue command gets them.

Reply:
im sorry but i am lost on how the whole config thing works. Might be because im not very experienced in python, but i don't get it. I added the try to get the config, but i'd appreciate your help on implementing the prefixes.
Do you want the repo param to be a the prefix?